Maestro/subscriptions.json

```diff
@@ -1165,55 +1163,39 @@
         }
       }
     },
-    // Automate opening PRs to merge aspnet release/3.0 branches back to release/3.1
-    // {
-    //   "triggerPaths": [
-    //     "https://github.com/aspnet/AspNetCore/blob/release/3.0//**/*",
-    //     "https://github.com/aspnet/AspNetCore-Tooling/blob/release/3.0//**/*",
-    //     "https://github.com/aspnet/Blazor/blob/release/3.0//**/*",
-    //     "https://github.com/aspnet/EntityFrameworkCore/blob/release/3.0//**/*",
-    //     "https://github.com/aspnet/Extensions/blob/release/3.0//**/*"
-    //   ],
-    //   "action": "github-dnceng-branch-merge-pr-generator",
-    //   "actionArguments": {
-    //     "vsoSourceBranch": "master",
-    //     "vsoBuildParameters": {
-    //       "GithubRepoOwner": "aspnet",
-    //       "GithubRepoName": "<trigger-repo>",
-    //       "HeadBranch": "<trigger-branch>",
-    //       "BaseBranch": "release/3.1"
-    //     }
-    //   }
-    // },
-    // {
-    //   "triggerPaths": [
-    //     "https://github.com/aspnet/EntityFramework6/blob/release/6.3//**/*"
-    //   ],
-    //   "action": "github-dnceng-branch-merge-pr-generator",
-    //   "actionArguments": {
-    //     "vsoSourceBranch": "master",
-    //     "vsoBuildParameters": {
-    //       "GithubRepoOwner": "aspnet",
-    //       "GithubRepoName": "<trigger-repo>",
-    //       "HeadBranch": "<trigger-branch>",
-    //       "BaseBranch": "release/6.4"
-    //     }
-    //   }
-    // },
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/aspnetcore/blob/release/2.1/**/*",
+        "https://github.com/dotnet/efcore/blob/release/2.1/**/*",
+        "https://github.com/dotnet/extensions/blob/release/2.1/**/*",
+      ],
+      "action": "github-dnceng-branch-merge-pr-generator",
+      "actionArguments": {
+        "vsoSourceBranch": "master",
+        "vsoBuildParameters": {
+          "GithubRepoOwner": "dotnet",
+          "GithubRepoName": "<trigger-repo>",
+          "HeadBranch": "<trigger-branch>",
+          "BaseBranch": "release/3.0"
+        }
+      }
+    },
+    // Reviewers: Could someone remind me why we don't do automatic merges through 'blazor-wasm' in aspnetcore-tooling and blazor?
```

Comment:
@mkArtakMSFT I'm starting to understand this: The 'blazor-wasm' branch doesn't exist in the aspnetcore-tooling or blazor repos. Please confirm that's as it should be.

Reply:
This sounds correct to me, that's only a branch in the aspnetcore repo. The blazor repo is just using master. cc @pranavkm @javiercn to confirm.